common/djangoapps/student/views.py

```diff
@@ -2304,7 +2309,7 @@ def do_email_change_request(user, new_email, activation_key=None):
 
     message = render_to_string('emails/email_change.txt', context)
 
-    from_address = microsite.get_value(
+    from_address = theming_helpers.get_value(
```

Comment:
old implementation was also using the same `email_from_address`


##### note: theming_helpers.get_value function further calling the `microsite.get_value`